README.md

```diff
@@ -1886,18 +1886,21 @@ npm run lint
 
 ### Modules
 
-- [12.14](#12.14) <a name="12.14"></a> Always use modules (`import`/ `export`) over a non-standard module system (CommonJS being the most popular of these).
+- [12.14](#12.14) <a name="12.14"></a> Always use modules (`import`/ `export`) with named exports over a different/non-standard module system (CommonJS being the most popular of these).
 
-  > Why? Modules are the future, so let’s get a head start. You can always transpile to a preferred module system.
+  Modules are the future, so let’s get a head start. You can always transpile to a preferred module system.
+
+  Avoid using `default` exports, consistently using named exports helps to keep naming and import style consistent across our repos.
 
   ```js
   // bad
   const BadImport = require('./BadImport');
   module.exports = BadImport.feelBadAboutIt();
+  export default BadImport();
```

Comment:
Is having cjs and esm style exports in a single example confusing as this is invalid syntax?

Reply:
The parens were already there, I just pushed up a commit removing them.

Does it read any better to you now?